cssdb.json

```
@@ -373,6 +373,15 @@
       }
     ]
   },
+  {
+    "id": "in-out-of-range",
+    "title": "`:in-range` and `:out-of-range` Pseudo-Classes",
+    "description": "If a temporal or number <input> has max and/or min attributes, then :in-range matches when the value is within the specified range and :out-of-range matches when the value is outside the specified range.",
```

Could we shorten the description to follow the _{TYPE} for {REASON}_? When paired with your examples, I think this will more clearly communicate this feature.

```suggestion
    "description": "A pseudo-class for matching elements that have range limitations,
```

---

Good suggestion _{TYPE} for {REASON}_
It would be good to add this into the CONTRIBUTING.md